pyls/workspace.py

```diff
@@ -267,6 +267,6 @@ def jedi_script(self, position=None):
 
             # Normalise the position as per the LSP that accepts character positions > line length
             # https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#position
-            line_len = len(self.lines[position['line']])
-            kwargs['column'] = min(position['character'], line_len - 1)
+            max_column = len(self.lines[position['line']]) - 1 if len(self.lines) > position['line'] else 0
```

Comment:
Doesn't this also end up with passing 0 into Jedi? Is there a test you could add perhaps?

Reply:
> Doesn't this also end up with passing 0 into Jedi?

It does, but does Jedi have a problem when 0 is passed to it?

This fix only prevents `IndexError: list index out of range` on line 270 because `self.lines` may be an empty list.

Sorry for not adding tests, I can add them later this week.

Reply:
OK, I added a small test.